consumers/email-consumer/main.py

```
@@ -0,0 +1,60 @@
+import json
+import os
+
+import requests
+from confluent_kafka import Consumer
+
+from lib.generate_html_verification_email import generate
+
+
+def send_verification_email(
+    email: str, verification_code: str, username: str, user_id: str
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)